config/pipeline.yaml

```diff
@@ -296,6 +313,8 @@ platforms:
       cpu: qemu64
       guestfs_interface: ide
 
+  qemu: *qemu-device
```

Comment:
Why are you redefining it? You could just use `qemu-x86` in the new scheduler entry

Reply:
I needed to redefine it as the device type name is `qemu` in the lab. There is no `qemu-x86` exists.
See https://lava.baylibre.com/scheduler/device_types?page=2&search=&length=100#table

Reply:
Of course, stupid me...